registry/test/podman.go

```diff
@@ -1,5 +1,5 @@
 /*
```

Comment:
maybe rename the path to test/registry would make more sense.

Reply:
ok